testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -36,8 +36,11 @@
   begin
     Timeout.timeout(DEFAULT_TIMEOUT) do
       loop do
-        break if page.has_content?(text)
-        sleep 3
+        begin
+          break if page.has_content?(text)
+        rescue Capybara::ElementNotFound
+          sleep 3
```

Comment:
That's wrong, sleep 3 should be at loop level, not only when an exception is raised.

Reply:
oops, fixed